examples/oidc/docker-compose.yml

```diff
@@ -56,5 +56,15 @@ services:
     image: devopsfaith/krakend:latest
     ports:
       - "8090:8090"
+      - "1234:1234"
     volumes:
       - ./krakend/krakend.json:/etc/krakend/krakend.json
+
+  jwt-revoker:
+    depends_on:
+      - krakend
+    container_name: jwt-revoker
+    build:
+      context: ./jwt-revoker
+    ports:
+      - "9000:8080"
```

Comment:
Correct me if I'm wrong: we should not expose this port to the outside. Please do not bind it...